controllers/apischeme/apischeme_controller.go

```diff
@@ -94,19 +70,22 @@ type LoadBalancer struct {
 	BaseDomain   string // What is the base domain (DNS zone) for the EndpointName record?
 }
 
-// Reconcile will ensure that the rh-api management api endpoint is created and ready.
```

Comment:
Are these old comments still relevant?

Reply:
Thanks, all the comments for the controllers are added back.